aspnetcore/data/ef-rp/intro/samples/cu/Pages/Students/Create.cshtml.cs

```diff
@@ -33,17 +30,38 @@ public IActionResult OnGet()
         [BindProperty]
         public Student Student { get; set; }
 
+        #region snippet_OnPostAsync
         public async Task<IActionResult> OnPostAsync()
         {
             if (!ModelState.IsValid)
             {
                 return Page();
             }
 
-            _context.Students.Add(Student);
-            await _context.SaveChangesAsync();
+            #region snippet_TryUpdateModelAsync
+            var studentToUpdate = new Student();
+            if (await TryUpdateModelAsync<Student>(
```

Comment:
Just to make sure, is there something happening asynchronously here? Otherwise, is there a non-async version of the method we can use?